docs/cosmos-hub/installation.md

```diff
@@ -9,8 +9,8 @@ Install `go` by following the [official docs](https://golang.org/doc/install). R
 ```bash
 mkdir -p $HOME/go/bin
 echo "export GOPATH=$HOME/go" >> ~/.bash_profile
-echo "export GOBIN=$GOPATH/bin" >> ~/.bash_profile
-echo "export PATH=$PATH:$GOBIN" >> ~/.bash_profile
+echo "export GOBIN=\$GOPATH/bin" >> ~/.bash_profile
```

Comment:
Why do we need `\` prefixes?

Reply:
In bash shell, variable in `""` would be extended.  The first version of cosmos docs was:
```bash
echo "export GOPATH=$HOME/go" >> ~/.bash_profile
source ~/.bash_profile
echo "export GOBIN=$GOPATH/bin" >> ~/.bash_profile
source ~/.bash_profile
echo "export PATH=$PATH:$GOBIN" >> ~/.bash_profile
source ~/.bash_profile
```

After `source ~/.bash_profile`, variable which was export just now was extended in current shell. So `echo "export GOBIN=$GOPATH/bin" >> ~/.bash_profile`  was extended to `echo "export GOBIN=/home/alice/go/bin" >> ~/.bash_profile`. Otherwise it would be extended to `echo "export GOBIN=/bin" >> ~/.bash_profile`.

After executing all steps in first version, we got `~/.bash_profile` :
```bash
......
export GOPATH=/home/alice/go
export GOBIN=/home/alice/go/bin
export PATH=/usr/bin/:usr/local/bin:......:/home/alice/go/bin
```

After executing all steps in current version, we got `~/.bash_profile` :
```bash
......
export GOPATH=/home/alice/go
export GOBIN=/bin
export PATH=/usr/bin/:usr/local/bin:......:/bin
```

After executing all steps in PR version, we got `~/.bash_profile` :
```bash
......
export GOPATH=/home/alice/go
export GOBIN=$GOPATH/bin
export PATH=$PATH:$GOBIN
```

We also can modify as following because variables in `''` would not be extended.
```bash
mkdir -p $HOME/go/bin
echo 'export GOPATH=$HOME/go' >> ~/.bash_profile
echo 'export GOBIN=$GOPATH/bin' >> ~/.bash_profile
echo 'export PATH=$PATH:$GOBIN' >> ~/.bash_profile
source ~/.bash_profile
```
Choosing prefix `\` for keeping pace with other docs.

Reply:
In all honestly, I prefer the single-quote syntax

Reply:
I like single-quote too. @alessio